.github/workflows/gitlab-ci.yaml

```
@@ -0,0 +1,86 @@
+      allow_failure: true
+  services:
+    - docker:dind
+  script:
+    - pip install pip-tools
+    - pip-compile --output-file requirements.tmp.txt requirements.in
+    - cat requirements.tmp.txt
+    - echo "take Diff:"
+    - diff --ignore-matching-lines=pip-compile --ignore-all-space requirements.txt requirements.tmp.txt
+  allow_failure: true
+
+test_build_image:
```

actionlint (1.7.4)

unexpected key "test_build_image" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)